lib/core/src/client/preview/start.js

```diff
@@ -118,13 +122,17 @@ export default function start(render, { decorateStory } = {}) {
   let previousKind = '';
   let previousStory = '';
   let previousRevision = -1;
+  let previousError;
 
   const renderMain = forceRender => {
     const revision = storyStore.getRevision();
     const selection = storyStore.getSelection();
-    const { kind, name, getDecorated, id } = selection || {};
+    const loadError = storyStore.getError();
+    const { kind, name, getDecorated, id, error } = selection || {};
 
-    if (getDecorated) {
+    if (loadError || error || (revision === previousRevision && previousError)) {
```

Comment:
What is the situation where `previousError` is defined but `error` is not?

Reply:
I don't even see `previousError` getting set anywhere...?

Reply:
good catch, will fix this